node/lib/ThingAccessor.ts

```diff
@@ -1,53 +1,53 @@
 
-import { DeviceInterface } from "./DeviceInterface";
-import { ITranslator } from "./ITranslator";
+import { ThingSchema } from "./ThingSchema";
+import { IThingTranslator } from "./IThingTranslator";
 
 import { EventEmitter } from "events";
 
 /**
- * Provides reflection-style access to device properties and methods via device interfaces.
+ * Provides reflection-style access to thing properties and methods via translators.
  */
-export class DeviceAccessor {
+export class ThingAccessor {
```

Comment:
@jeanpa thinks we need to discuss this name further. I will open a separate issue, does not block this PR for now.